ST_params.c

```diff
@@ -1000,6 +1025,46 @@ static void _rgroup_add2( char name[],
 }
 
 
+static void _rgroup_add_disturbance( char name[], Int killyr, Int killfreq_startyr, RealF killfreq,
+                      Int extirp, Int mort, RealF prop_killed, RealF prop_recovered,RealF grazing_frq,RealF prop_grazing, Int grazingfreq_startyr) {
+/*======================================================*/
+  GrpIndex rg;
+  
+   char name2[80];
+
+   _setNameLen(name2, name, MAX_GROUPNAMELEN);
+   rg = RGroup_Name2Index( name2);
+   if (rg <0) {
+     LogError(logfp, LOGFATAL, "%s: Mismatched name (%s) for disturbance",
+             MyFileName, name2);
+   }
+
+  RGroup[rg]->killyr        = killyr;
+  RGroup[rg]->killfreq_startyr = killfreq_startyr;
+  RGroup[rg]->killfreq      = killfreq;
+  RGroup[rg]->extirp        = (IntS) extirp;
+  RGroup[rg]->use_mort      = itob(mort);
```

Comment:
if possible - can we move `mort` out? Here and elsewhere - see comment above...

Reply:
@kpalmqui  I can move `mort` from `_rgroup_add_disturbance` to `_rgroup_add1` if that works.

Reply:
use_mort is now set in `_rgroup_add1`. Let me know if this is what you were looking for!